subsys/net/ip/tcp.c

```diff
@@ -2931,7 +2937,7 @@ static enum net_verdict tcp_in(struct tcp *conn, struct net_pkt *pkt)
 			conn_ack(conn, th_seq(th) + 1); /* capture peer's isn */
 			tcp_out(conn, SYN | ACK);
 			conn->send_options.mss_found = false;
-			conn_seq(conn, + 1);
+			conn_seq(conn, 1);
```

Comment:
That's similar to https://github.com/zephyrproject-rtos/zephyr/pull/89862#discussion_r2086132837, same in other places below.